ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Validators/CreateProjectDTOValidator.cs

```diff
@@ -21,8 +35,20 @@ public CreateProjectDTOValidator()
                 .NotEmpty()
                 .MaximumLength(50);
 
+            RuleFor(p => p.EmployeeIds)
+                .MustAsync(async (ids, cancellation) =>
+                {
+                    bool exists = await _userservice.ExistAsync(ids, cancellation);
+                    return exists;
+                });
+
             RuleForEach(p => p.EmployeeIds)
-                .NotEmpty();
+                .NotEmpty()
+                .MustAsync(async (id, cancellation) =>
```

Comment:
you already checked that all ids from EmployeeIds were found (counts are same) so it's not necessary to check it one more time one by one